### PR TITLE
Show election in the navbar

### PR DESCRIPTION
**Pull request by oliver3**
Fixes https://github.com/kiesraad/abacus/pull/453#discussion_r1804747108
- Added clickable election name on `/elections/1/polling-stations`
- Added election name on `/elections/1`
- Remove padding on first link in the navbar to align left with the rest of the page
image

### CODE REVIEW

**Review by jschuurk-kr — APPROVED** [review]

**Review by jorisleker — CHANGES_REQUESTED** [review]
We already have `elections.location` in our database, which is one of the attributes that should be shown in this navbar element as well. Could you add this? 

image

The element is constructed as follows:
- regional scope / role of polling station in this election (GSB/HSB/CSB); not in database yet, implemented later
- code / number of the region (gemeentenummer, kieskring); not in database yet; implemented later
- name of the region (gemeente, kieskring, provincie, waterschap); `elections.location`
- name of the election: `elections.name`. 

All items together are 1 link to `/elections/:election_id`

**Review by Lionqueen94 — APPROVED** [review]

**Comment by oliver3** [comment]
Thanks for the thorough clarification @jorisleker ! I've added the location

**Review by jorisleker — DISMISSED** [review]
LGTM!

File: frontend/app/module/data_entry/polling_station/PollingStationLayout.test.tsx [hunk 1]
```diff
@@ -50,7 +50,7 @@ describe("PollingStationLayout", () => {
     // Check if the navigation bar displays the correct information
     const nav = await screen.findByRole("navigation", { name: /primary-navigation/i });
     expect(within(nav).getByRole("link", { name: "Overzicht" })).toHaveAttribute("href", "/elections");
-    expect(within(nav).getByRole("link", { name: election.name })).toHaveAttribute(
+    expect(within(nav).getByRole("link", { name: new RegExp(election.name) })).toHaveAttribute(
```
Comment (jschuurk-kr) [inline frontend/app/module/data_entry/polling_station/PollingStationLayout.test.tsx:53]:
```suggestion
    expect(within(nav).getByRole("link", { name: /election.name/ })).toHaveAttribute(
```

RTL [documentation has](https://testing-library.com/docs/queries/about#textmatch) this way of doing regex. We also do it that way in some other tests.

Comment (jschuurk-kr) [reply frontend/app/module/data_entry/polling_station/PollingStationLayout.test.tsx:53]: never mind, bad idea with variables

**Review by jschuurk-kr — APPROVED** [review]

**Pull request merged**